extensions/wikia/ImageReview/ImageReviewEventsHooks.class.php

```diff
@@ -27,6 +43,8 @@ public static function onArticleUndelete( Title $title, $created, $comment ) {
 	public static function onArticleDeleteComplete( Page $page, User $user, $reason, $articleId ) {
 		global $wgCityId;
 
+		// TODO: should ImageReview service be notified about article delete?
```

Comment:
That would be "nice to have".  If there is a possibility to delete a task from queue so the service does not have to process it, we should do that.